src/util/time.cpp

```diff
@@ -75,6 +75,13 @@ void MockableSteadyClock::ClearMockTime()
 
 int64_t GetTime() { return GetTime<std::chrono::seconds>().count(); }
 
+int64_t GetTimeMillis()
```

Comment:
I doubt we'll be restoring this. It was removed intentionally.

Reply:
yes, I'm going to remove this, will push an update shortly